tests/test_radio_dish.cpp

```diff
@@ -395,15 +395,14 @@ static bool is_multicast_available (int ipv6_)
 
     msleep (SETTLE_TIME);
 
-    rc = sendto (send_sock, msg, static_cast<socklen_t> (strlen (msg)), 0,
-                 &mcast.generic, sl);
+    rc = sendto (send_sock, msg, strlen (msg), 0, &mcast.generic, sl);
     if (rc < 0) {
         goto out;
     }
 
     msleep (SETTLE_TIME);
 
-    rc = recvfrom (bind_sock, buf, sizeof (buf) - 1, 0, NULL, 0);
+    rc = recvfrom (bind_sock, buf, sizeof (buf) - 1, MSG_DONTWAIT, NULL, 0);
```

Comment:
ZMQ_DONTWAIT?

Reply:
No, this is a call to recvfrom which is a system call. It has no knowledge of ZMQ API constants.
ZMQ_DONTWAIT is for ZMQ API calls lik zmq_msg_recv and zmq_msg_send.